app/models/contact.rb

```diff
@@ -1,3 +1,5 @@
+require 'deserializers/xml/legal_document'
```

Comment:
Maybe we should include deserializers in config.autoload_paths so we don't need to require them? But for sure it's up to you, just wondering.